ansible/roles/bastion/tasks/k8s.yml

```diff
@@ -0,0 +1,18 @@
+---
+- tags:
+    - bastion_k8s
+  block:
+  # TODO: (jbirchler) Ask JimR if this should be deleted (it is deleted upstream)
```

Comment:
I'm not sure what my own TODO here is about. This appears in Jim's fork as an addition.

Reply:
Look to see when this was deleted and what else was in the commit/PR with the deletion.

Reply:
This might solve the "fixups" hack